server/src/main/scala/com/metamx/tranquility/server/PropertiesBasedServerConfig.scala

```diff
@@ -19,6 +19,10 @@
 
 package com.metamx.tranquility.server
 
+import java.security.KeyStore
+import java.util
```

Comment:
This makes the code look a bit weird, imo. I usually prefer `import java.{util => ju}` and then using `ju.List` etc.

Reply:
Changed this to `ju.List`